packages/nextra/package.json

```diff
@@ -144,6 +144,9 @@
     "gray-matter": "^4.0.3",
     "hast-util-to-estree": "^3.1.0",
     "katex": "^0.16.9",
+    "mdast-util-from-markdown": "^2.0.1",
+    "mdast-util-gfm": "^3.0.0",
+    "mdast-util-to-hast": "^13.2.0",
```

Comment:
these dependencies are added to parse JSDoc description? since it can contain markdown right?